l10n_ro_account_edi_ubl/models/account_edi_xml_cius_ro.py

```diff
@@ -46,6 +46,14 @@ def _get_partner_party_tax_scheme_vals_list(self, partner, role):
                 vals["tax_scheme_id"] = "!= VAT"
         return vals_list
 
+    def _get_partner_party_legal_entity_vals_list(self, partner):
+        val_list = super()._get_partner_party_legal_entity_vals_list(partner)
+        if not partner.is_company and not partner.parent_id and not partner.vat:
```

Comment:
Cred ca ar trebui pus un camp pe partener daca se trimite cnp-ul la persoane sau nu, de obicei fara un camp de cnp in baza folosim vat pentru persoane fizice, deci daca e completat trebuie sa avem posibilitatea sa ii dam clinetului sa aleaga daca vrea sa ii trimitem pe cnp sau nu.

Reply:
Cazul in care la CUI e completat cu CNP pentru a fi utilizat in at scop este unul particular.

Reply:
Atata timp cat si Odoo trateaza la fel VAT ca si CNP in SAFT cred ca trebuie sa luam acest lucru in calcul